include/variables.h

```diff
@@ -2742,7 +2742,7 @@ extern f32 D_801DE860;
 extern f32 D_801DE864;
 extern f32 D_801DE868;
 extern f32 D_801DE884;
-extern TexturePtr D_801DE890[];
+extern TexturePtr gLensOfTruthMaskTex[];
```

Comment:
This doesn't seem like an array of `TexturePtr`s? Especially with the name you are giving it.